python-wrapper/src/neo4j_viz/__init__.py

```diff
@@ -1,6 +1,25 @@
 from .node import Node
-from .options import CaptionAlignment, Layout, Renderer
+from .options import (
+    CaptionAlignment,
+    Direction,
+    ForceDirectedLayoutOptions,
+    HierarchicalLayoutOptions,
+    Layout,
+    Packing,
+    Renderer,
+)
 from .relationship import Relationship
 from .visualization_graph import VisualizationGraph
 
-__all__ = ["VisualizationGraph", "Node", "Relationship", "CaptionAlignment", "Layout", "Renderer"]
+__all__ = [
+    "VisualizationGraph",
+    "Node",
+    "Relationship",
+    "CaptionAlignment",
+    "Layout",
+    "Renderer",
+    "HierarchicalLayoutOptions",
+    "ForceDirectedLayoutOptions",
```

Comment:
In general I'm against introducing smaller objects like this an API. I find it to be a bit more overwhelming for the user to have to learn about and import these classes, instead of just letting them provide a dict. Internally it's great to have though

Reply:
True, we could allow both types. But that also makes the docs look complicated 😅